tests/Feature/Connect/UploadAchievementTest.php

```diff
@@ -395,7 +395,7 @@ public function testJrDevPermissions(): void
         $game = $this->seedGame(withHash: false);
 
         /** @var Achievement $achievement1 */
-        $achievement1 = Achievement::factory()->create(['GameID' => $game->ID, 'user_id' => $this->user->id]);
+        $achievement1 = Achievement::factory()->create(['GameID' => $game->ID, 'user_id' => $this->user->id, 'Points' => 5]);
```

Comment:
Logic below makes sure this value isn't 10 after a failed update. It could randomly be 10 from the factory now (could only by up to 9 before because of the indexing error).